### ISSUE TITLE
how to install the latest version

### ISSUE THREAD

**Issue by cannonli7** [zhanghang1989/PyTorch-Encoding#394]
I install the package with command line"pip install torch-encoding" and found the version is 1.2.1, how to install the latest version because I found that there must be some files missing under version 1.2.1

**Comment by xialang2012** [comment]
Did you install the Pytorch-Encoding well?

**Comment by cannonli7** [comment]
Yes, I suggest installing by command " pip install git+https://github.com/zhanghang1989/PyTorch-Encoding/ ", you may need to try multiple times. Once you succeed, go to anaconda3/envs/.../site-packages/encoding/ , copy some .py files you need from folder Pytorch-Encoding(such as folder "dilated", log.py, lr_scheduler.py from folder "utils")



At 2021-05-25 17:40:41, "xialang2012" ***@***.***> wrote:

Did you install the Pytorch-Encoding well?
